### DISCUSSION TITLE
Dragon's Dogma

### DISCUSSION THREAD

**Discussion by TheMercenary333** [skyline-emu/skyline#1479] [Game Requests]
![H2x1_NSwitchDS_DragonsDogmaDarkArisen_image1600w](https://user-images.githubusercontent.com/74915480/172027488-c43b394d-5c10-46f0-a49a-097378974ae1.jpg)

**Comment by Crytonics** [comment]
Game is on list: https://github.com/skyline-emu/skyline-games-list/issues/176
